src/server.py

```diff
@@ -458,12 +459,15 @@ def run_seta_details_query():
     date = request.args.get("date")
     db = create_db_connnection()
     cursor = db.cursor()
-    query = "select jobtype from seta where date='%s'" % date
+    query = "select jobtype from seta where date='%s 00:00:00'" % date
```

Comment:
this is actually the format it is in for the database.  I could do a followup and force this to just a yyyy-mm-dd and drop the 00:00:00 from the database and all related code.